src/components/calculator.js

```diff
@@ -0,0 +1,71 @@
+function mainCalculator() {
```

Comment:
-  [ ] This activity, we want the _Calculator_ component to be class-based component. Kindly improve upon this aspect.